clusterloader2/pkg/util/util.go

```diff
@@ -24,50 +24,102 @@ import (
 	"time"
 )
 
+var errKeyNotFound = fmt.Errorf("key not found")
+
 // GetString tries to return value from map cast to string type. If value doesn't exist, error is returned.
 func GetString(dict map[string]interface{}, key string) (string, error) {
-	return getString(dict, key, "", fmt.Errorf("key %s not found", key))
+	value, err := getString(dict, key)
+	if err == errKeyNotFound {
```

Comment:
If you create the type I suggested, you won't need all of those strange iffs.

Reply:
I got rid of half of the ifs.
I think ifs for handling the default values are still needed. WYT?